plutus-wallet-api/ledger/Ledger/Validation.hs

```diff
@@ -400,31 +394,34 @@ signsTransaction (Signature sig) (PubKey (LedgerBytes pk)) p =
 {-# INLINABLE valueForged #-}
 -- | Value forged by a 'PendingTx'.
 valueForged :: PendingTx -> Value
-valueForged (PendingTx _ _ _ forge _ _ _ _) = forge
+valueForged = pendingTxForge
```

Comment:
Delete?

Reply:
I mean, maybe you want to keep these around since they're semantically more meaningful?